client/checkout/blocks/payment-method-label.js

```diff
@@ -57,6 +57,8 @@ export default ( { api, title, countries, iconLight, iconDark, upeName } ) => {
 		getUPEConfig( 'wcBlocksUPEAppearanceTheme' )
 	);
 
+	const fontRules = useMemo( () => getFontRulesFromPage(), [] );
```

Comment:
```suggestion
	const fontRules = useMemo( getFontRulesFromPage, [] );
```
Not sure if this is even an enhancement or if this is unconventional and might have longer term drawbacks, if we decide to add some parameters to `getFontRulesFromPage`, but since `getFontRulesFromPage` currently requires no function parameters, this could save us a pocket-full of characters. 🤷 

Feel free to ignore, as perhaps leaving this as it is currently is more conventional for usage with `useMemo`.

Reply:
Thanks for the suggestion but I think I'll just keep the current implementation as it is more explicit and conventional, as you mentioned.